QuantumRangers-SE-Team-10/lasertech
# Team Score Display

### Summary
In this PR we add the team score display to the player playerDisplay screen

### Details
I created a new file and called it teamScoreDisplay along with the associated css module file. The TeamScoreDisplay class takes in the playerSession as props which is an array of player session. The useEffect monitor for changes in the playerSession props and re-calculate the team score and update the state variable.

### Design Decision
I decided to separate this component into a separate file because its cleaner and more readable.  

### Testing Instruction
1. add this code to PlayerDisplay file 
```
useEffect(() => {
    if(!redPlayers[0]) return 

    setTimeout(() => {
      setRedPlayers(old =>{
        const temp = [...old]
        temp[0].playerScore += 1
        return temp
      })
    }, 1000)
  },[redPlayers]) 
````
![image](https://github.com/QuantumRangers-SE-Team-10/lasertech-frontend/assets/76791231/275248cd-ed80-4a11-a549-386919d40430)

2. navigate to http://localhost:5173/game?id=133
3. If everything work, Tyler score should increment by 1 every seconds and the team score should also be updated.
![image](https://github.com/QuantumRangers-SE-Team-10/lasertech-frontend/assets/76791231/33c42292-5e2b-433f-a822-41e4d3c2703c)

### Checks
